src/main/java/ee/tuleva/onboarding/fund/statistics/PensionFundStatisticsResponse.java

```diff
@@ -0,0 +1,19 @@
+package ee.tuleva.onboarding.fund.statistics;
+
+import lombok.Data;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
+
+@Data
+@XmlRootElement(name = "RESPONSE", namespace = "http://corporate.epis.ee/producer/")
+@XmlAccessorType(XmlAccessType.FIELD)
+public class PensionFundStatisticsResponse {
```

Comment:
I thought about it, but then saw that we use a lot of `*Request` and `*Response` class names, so I ended up following the "convention".